src/main/java/com/springCommerce/commerce/controller/UserController.java

```diff
@@ -0,0 +1,44 @@
+package com.springCommerce.commerce.controller;
+
+
+import com.springCommerce.commerce.dto.CreateUserRequest;
+import com.springCommerce.commerce.dto.UpdateUserRequest;
+import com.springCommerce.commerce.dto.UserDto;
+import com.springCommerce.commerce.model.User;
+import com.springCommerce.commerce.service.UserService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/user")
+public class UserController {
+
+    private final UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @GetMapping
```

Comment:
Did you forget to give a value? I mean something like `@GetMapping("/path")`

Reply:
I wanted to response to the get request that came via the '/v1/user' path.